src/recipe.jl

```diff
@@ -50,7 +50,8 @@ end
         pop!(bot)
     end
     hull = [top; bot]
+    hull_cyclic = [hull; hull[[1]]]
```

Comment:
You can do `push!(hull, hull[1])` to avoid new allocations

Reply:
I personally prefer the non-mutating style, but I've just force-pushed the change :)